code/modules/mob/living/simple_animal/aquatic/aquatic_carp.dm

```diff
@@ -6,13 +6,6 @@
 	maxHealth = 20
 	health = 20
 
-	meat_type = /obj/item/chems/food/fish/carp
```

Comment:
Unintentional change? `retaliate/aquatic` only has normal fish meat, not carp meat.